### PR TITLE
feat(orchestrator): execute workflow page polishing

### PR DESCRIPTION
**Pull request by batzionb**
Added auto focus to first field when entering each step in the form
Added auto focus to the Run button to Review and Run
Improved the page header to make the context more clear as agreed with UX
Improved the missing schema info message as agreed with UX
image
image

### CODE REVIEW

File: plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.stories.tsx [hunk 1]
```diff
@@ -35,6 +36,11 @@ const meta = {
                   ?.schemaResponse
                   ? context?.args?.schemaResponse()
                   : Promise.resolve(fakeDataInputSchemaReponse),
+                executeWorkflowResponse: () => {
+                  // eslint-disable-next-line no-alert
+                  alert('Execute workflow API called');
```
[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.stories.tsx:41]: It's just for storybook so when you click on Run you'll see that the backend call was invoked

File: plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx [hunk 2]
```diff
@@ -6,21 +6,32 @@ const SubmitButton = ({
...
-  >
-    {children}
-  </Button>
-);
+  focusOnMount?: boolean;
+}) => {
+  const ref = React.useRef<HTMLButtonElement>(null);
+  React.useEffect(() => {
+    if (focusOnMount) {
+      ref.current?.focus();
```
[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx:19]: Yeah you're right
For some reason it only happens if you click Enter on "Next step" but not if you it with the mouse...
I'm checking what can be done

[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx:19]: This is the L&F that comes with material ui for focus on primary buttons
So there isn't much to do about it

[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx:19]: As can be seen here when you move between the buttons you get this focus behavior

[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx:19]: Maybe in RHDH it'll look better if this is changed in the theme

[the remark this answers is not in the document]
Comment (caponetto) [reply plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx:19]: I see 😞

File: plugins/orchestrator/src/__fixtures__/fakeWorkflowDataInputSchemaResponseMultiStep.tsx
[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/__fixtures__/fakeWorkflowDataInputSchemaResponseMultiStep.tsx:1]: done

File: plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (batzionb) [reply plugins/orchestrator/src/components/SubmitButton/SubmitButton.tsx:19]: I still think it's better to have it focused like this than not at all

**Review by caponetto — APPROVED** [review]

**Pull request merged**